sql/ddl_api.sql

```diff
@@ -91,11 +91,29 @@ BEGIN
     WHERE attrelid = main_table AND attname = time_column_name;
 
     -- Timestamp types can use default value, integral should be an error if NULL
-    IF time_type IN ('TIMESTAMP', 'TIMESTAMPTZ', 'DATE') AND chunk_time_interval IS NULL THEN
-        chunk_time_interval_actual := _timescaledb_internal.interval_to_usec('1 month');
-    ELSIF time_type IN ('SMALLINT', 'INTEGER', 'BIGINT') AND chunk_time_interval IS NULL THEN
-        RAISE EXCEPTION 'chunk_time_interval needs to be explicitly set for types SMALLINT, INTEGER, and BIGINT'
-        USING ERRCODE = 'IO102';
+    IF time_type IN ('TIMESTAMP', 'TIMESTAMPTZ', 'DATE') THEN
+        IF chunk_time_interval IS NULL THEN
+            chunk_time_interval_actual := _timescaledb_internal.interval_to_usec('1 month');
+        ELSIF pg_typeof(chunk_time_interval) IN ('INT'::regtype, 'SMALLINT'::regtype, 'BIGINT'::regtype) THEN
+            chunk_time_interval_actual := chunk_time_interval::BIGINT;
```

Comment:
Should we enforce a minimum chunk interval here, even if its like 1 second (which is way smaller than I think you'd ever want)?

I mean this more to catch the bugs which we occasionally see that people specify chunk interval as '86400' instead of '86400000000' for one day.  Might be a good "defensive" check?

Reply:
Done. threw a warning not an error.